Twig/Extension/PhoneNumberHelperExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony2 PhoneNumberBundle.
+ *
+ * (c) University of Cambridge
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Misd\PhoneNumberBundle\Twig\Extension;
+
+use Misd\PhoneNumberBundle\Templating\Helper\PhoneNumberHelper;
+
+/**
+ * Phone number helper Twig extension.
+ */
+class PhoneNumberHelperExtension extends \Twig_Extension
+{
+    /**
+     * Phone number helper.
+     *
+     * @var PhoneNumberHelper
+     */
+    protected $helper;
+
+    /**
+     * Constructor.
+     *
+     * @param PhoneNumberHelper $helper Phone number helper.
+     */
+    public function __construct(PhoneNumberHelper $helper)
+    {
+        $this->helper = $helper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('phone_number_format', array($this->helper, 'format'), array('deprecated' => '1.2')),
+            new \Twig_SimpleFunction('phone_number_is_type', array($this->helper, 'isType'), array('deprecated' => '1.2')),
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return array(
+            new \Twig_SimpleFilter('phone_number_format', array($this->helper, 'format')),
+            new \Twig_SimpleFilter('phone_number_is_type', array($this->helper, 'isType')),
```

Comment:
Usually, this would rather be a test in Twig templates. You would then use it like this:

```twig
{% if value is phone_number_type(type) %}
    {# ... #}
{% endif %}
```

Reply:
@xabbuh the problem in your example is that `value` always has to be the content of the constant. But the parameter for `phone_number_is_type` can also be a name of a constant in ``\libphonenumber\PhoneNumberFormat``.
I did this the same as the usage of `format`. You can take a look at https://github.com/misd-service-development/phone-number-bundle/pull/57/files#diff-549733e4205d824d550c9163b73ce246R110

Reply:
@toooni Well, it doesn't matter if you need to pass values to a filter, function or test. Can you show a Twig template using your filter that you have in mind that would not be writable with the test?

Reply:
@xabbuh oh. I didn't know that theres a `\Twig_SimpleTest`. I'll make the changes.

Reply:
@xabbuh is it ok when i name the test `phone_number_of_type`? And leave the deprecated function (I'll remove the filter).

Reply:
I have no preference for the name (but I also don't have anything to decide here ;)).

Reply:
But you're right. That should be enough ;) I made the changes - we'll see.